### PR TITLE
Reenable `source-kafka` integration tests

### PR DESCRIPTION
**Pull request by saterus**
These were not turned off for any particular reason. I believe it was left out of a bit of refactoring of the `ci.yaml` while two branches were in flight. A logical merge conflict.

---
This change is [Reviewable](https://reviewable.io/reviews/estuary/connectors/42)

### CODE REVIEW

**Review by jgraettinger — APPROVED** [review]
LGTM

**Comment by jgraettinger** [comment]
Sorry if I removed this during a merge. Thinking that's probably what happened...

File: source-kafka/src/state.rs [hunk 1]
```diff
@@ -89,38 +89,49 @@ impl<'m> From<&BorrowedMessage<'m>> for Offset {
...
-            .expect("the topic must be configured before checkpointing");
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize)]
+pub struct CheckpointSet(pub BTreeMap<String, BTreeMap<i32, Offset>>);
+
+impl CheckpointSet {
+    pub fn add(&mut self, new_checkpoint: Checkpoint) {
+        use std::collections::btree_map::Entry;
 
-        topic.offset = updated_topic.offset;
+        match self.0.entry(new_checkpoint.topic) {
```
Comment (psFried) [inline source-kafka/src/state.rs:125]: super nit:

```suggestion
       self.0.entry(new_checkpoint.topic).or_insert_with(Default::default).insert(new_checkpoint.partition, new_checkpoint.offset);
```

**Review by psFried — APPROVED** [review]
LGTM

**Review by jgraettinger — APPROVED** [review]
Not sure if you were waiting for me, but LGTM as well

> Reviewed 1 of 1 files at r1, 8 of 12 files at r2, 5 of 5 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/estuary/connectors/42#-:-Mo3x72Y7rKxNM1YJ__-:b-4aexgn)* status: all files reviewed, 1 unresolved discussion (waiting on @saterus)

**Comment by jgraettinger** [comment]
Is this ready to land ?

**Pull request merged**